src/client/containers/StudentHome/StudentHome.js

```diff
@@ -0,0 +1,83 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { Layout } from '../../components/Layout';
+import Loader from '../../components/Loader/Loader';
+import { AppHeader } from '../../components/Appheader/AppHeader.component';
+import { Avatar } from '../../components/Avatar/Avatar';
+import student from '../../assets/images/student.png';
+import { Card } from '../../components/Card/Card';
+import { CardWithEventsOrStudyGroups } from '../../components/CardWithEventsOrStudyGroups/CardWithEventsOrStudyGroups';
+import HYFLogo from '../../components/StudyGroupLogo/HYF.logo.svg';
+import ZendeskLogo from '../../components/StudyGroupLogo/Zendesk.logo.svg';
+import ElasticLogo from '../../components/StudyGroupLogo/Elastic.logo.svg';
+import Footer from '../../components/footer/footer';
+
+function StudentHome() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [events, setEvents] = useState([]);
+  const cardTitle = 'No events to show here';
+  const cardText =
+    'There are no events to show here maybe  you did not book any event yet. Please go to events page and book an event';
+  useEffect(() => {
+    axios.get('/api/events').then((response) => {
```

Comment:
This is the wrong URL, find out the correct URL by asking colleagues or going through closed issues. This should be a list of events that the user has applied for.

Reply:
yes true,  this is the URL 
​/eventsUsers​/{userID}​/{role}.  can you give me an idea to get username. if i get user name i can get userId and role. i am a bit confused in this authentication part